eng/pipelines/runtimelab-official.yml

```diff
@@ -46,7 +46,7 @@ extends:
               templatePath: 'templates-official'
               isOfficialBuild: true
               timeoutInMinutes: 180
-              buildArgs: -s clr+libs+hosts+packs -c $(_BuildConfig)
+              buildArgs: -s clr+libs+clr.hosts+packs -c $(_BuildConfig)
```

Comment:
Could you please backport these fixes to dotnet/runtime so that the next person setting up an runtimelab experiment gets them?

Reply:
Yes, I am planning on it.